src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/controller/isl/IslFsm.java

```diff
@@ -135,7 +138,7 @@ public void operationalEnter(IslFsmState from, IslFsmState to, IslFsmEvent event
         monitorsByPriority = ImmutableList.of(
                 new DiscoveryMovedMonitor(reference),
                 new DiscoveryPortStatusMonitor(reference),
-                new DiscoveryBfdMonitor(reference),
+                discoveryBfdMonitor,
```

Comment:
No... the idea here is to reset internal monitor state (ISL can reenter operational state), so you must not use any global monitor instances, you must create all of them here.

Reply:
Fixed.